content/admin/policy_mgmt/IPManagement.md

```diff
@@ -4,8 +4,9 @@ description: "Overview of IP management"
 tags: "Admin IP policy management protect"
 -->
 
-IP Management is Contrast's blacklist and whitelist (trusted hosts) management feature. Organization and Policy (Rules) have access to this feature. Go to **Policy Management** from the user menu and select IP Management from the left navigation. Here you can see the current hosts on the blacklist and whitelist as well as manage new and existing hosts. You can specify which servers you’d like the rule to apply to and set an expiration on the rule in the event you just want to place a host in a “penalty box”.
+Manage IPs in your organization with blacklists, whitelists (trusted hosts) and source names. Organization and Rules admins can go to the **user menu > Policy Management > IP Management tab** to create and manage your preferences.
 
-* **IP Blacklists** allow you to put a rule in place to distrust an IP, considering it unacceptable. This is used for immediate triage until a more permanent protection policy can be put in place or investigation can be conducted.  
-* **Trusted Hosts** can be used to whitelist a host that is conducting an internal vulnerability scan to avoid polluting Contrast with non-attack data.  
+* **IP Blacklists** allow you to put rules in place to distrust IP addresses, which are then considered to be unacceptable. This is appropriate for immediate triage until you can put a more permanent Protect policy in place or conduct an investigation.
+* **IP Whitelists** let you mark trusted hosts conducting internal vulnerability scans as safe to avoid polluting Contrast with non-attack data. Data for whitelisted IP addresses does not appear in the Contrast UI. 
+* [Source Names](admin-ipmgmt.html#source-name) allow you to label one or more trusted IP addresses or subnet masks with any name you choose. The chosen name then appears in your **Attacks > Monitor** as well as **Attack Details** pages, which allows you to quickly identify and differentiate events by these users from attack events that need your attention. 
```

Comment:
I took another crack at this paragraph. Hopefully it captured the all of the necessary changes; if not, I can try again! 

For the future updates note, I think we should save that for the release notes and What's New. We generally don't put temporary or time-sensitive alerts in OpenDocs, and I'd rather not make an exception in this case unless it would be exceptionally helpful.

Reply:
Ok makes sense about the future updates. Looking for the updated paragraph and will make a note when I find it. :)